tests/e2e/specs/mailchimp.test.js

```diff
@@ -23,13 +19,18 @@ async function activatePublicizeModule() {
 }
 
 describe( 'Mailchimp Block', () => {
+	beforeAll( async () => {
+		await resetWordpressInstall();
```

Comment:
Would it be enough to reset site's plan instead?

Reply:
I think it depends :) 

This PR makes _all_ the tests completely independent, if we will reset site's plan only, we'll need to set up a connection before running any tests that expect that. 

Right now we have 4 test suites:
- 3 test suites for paid blocks
since jest running the tests in random order

Reply:
I think it depends :) 

This PR makes _all_ the tests completely independent, if we will reset site's plan only, we'll need to set up a connection before running any tests that expect that. 

Right now we have 4 test suites:
- 3 test suites for paid blocks
- 1 suite for pre-connection checks

since jest running the tests in random order, we can not guarantee the order in which these tests would be executed, meaning these test suites should be self-contain as much as possible (setting-up the preconditions, and cleaning up after them selfs)

So I'd say by resetting the site completely (btw, it also generates new site url) we'll be on the safe side